docs/source/tutorials/oss.rst

```diff
@@ -0,0 +1,80 @@
+Optimizer state sharding
+========================
+
+Using torch.nn.parallel.DistributedDataParallel leads to some wasted communications, but it is possible and makes OSS a drop in solution in your existing torch distributed code.
+Let's suppose that your trainer looks likemake html
+
+.. code-block:: default
+
+
+    import torch
+
+    def train(
+        rank: int,
+        world_size: int,
+        epochs: int):
+
+        # DDP
+        dist_init(rank, world_size)
+
+        # Problem statement
+        model = myAwesomeModel()
+        dataloader = mySuperFastDataloader()
+        loss = myVeryRelevantLoss()
+
+        base_optimizer_arguments = {} # pass any optimizer specific arguments here, or directly below when instantiating OSS
+        optimizer = torch.optim.SGD(params=model.parameters() **base_optimizer_arguments)
```

Comment:
typo?

Reply:
quick copy paste, not the nicest, I meant to minimize the changes in between the two configs. You mean about the missing comma (fixing that :/) or worse, that it's not very clear ?